src/Juvix/Core/IR/Typechecker.hs

```diff
@@ -73,12 +73,17 @@ typeTerm ∷
   Term primTy primVal →
   Annotation primTy primVal m →
   m ()
-
--- * (Universe formation rule)
+-- ★ (Universe formation rule)
 
 typeTerm _ _ii _g t@(Star i) ann = do
   typeTermIntroLog t ann
-  logOutput "patterned matched to be a * term. Type checker applies the universe formation rule. Checking that Sigma is zero. "
+  logOutput
```

Comment:
This is fine but the spacing at the end were intentional. I think having a white space between sentences are easier to read.

Reply:
Hmm, adding it here, seems rather error prone, as one can forget it. Could this be added to the logOutput function itself?

Reply:
That may be a good idea. Can be tweaked further...I expect I'll have to change the logs along the way.

Reply:
I was going to take them out as a new datatype as Chris suggested. But after we've debugged more.